src/app/image-slots/image-slots.component.html

```diff
@@ -0,0 +1,7 @@
+<section class="row justify-content-between image-wrapper">
+    <div class="img-box selected" id="beach"(mouseenter)="selectClass('beach')"><img ngSrc="https://picsum.photos/500" fill class="object-fit-cover"><i class="bi bi-star" ></i></div>
+    <div class="img-box" id="pools" (mouseenter)="selectClass('pools')"><img ngSrc="https://picsum.photos/500" fill class="object-fit-cover"><i class="bi bi-star"></i></div>
+    <div class="img-box" id="transport" (mouseenter)="selectClass('transport')"><img ngSrc="https://picsum.photos/500" fill class="object-fit-cover"><i class="bi bi-star"></i></div>
+    <div class="img-box" id="aquapark" (mouseenter)="selectClass('aquapark')"><img ngSrc="https://picsum.photos/500" fill class="object-fit-cover"><i class="bi bi-star"></i></div>
+    <div class="img-box" id="gardens" (mouseenter)="selectClass('gardens')"><img ngSrc="https://picsum.photos/500" fill class="object-fit-cover"><i class="bi bi-star"></i></div>
```

Comment:
Jest lepiej niż było, z tym że cały czas moim zdaniem można to poprawić, można wywalić te klasy w div'ie bo każda jest taka sama, to samo w img i w znaczniku i